django/core/validators.py

```diff
@@ -104,7 +107,101 @@ class URLValidator(RegexValidator):
         r'(?:[/?#][^\s]*)?'  # resource path
         r'\Z', re.IGNORECASE)
     message = _('Enter a valid URL.')
-    schemes = ['http', 'https', 'ftp', 'ftps']
+    schemes = [
```

Comment:
Can you explain how this is backwards incompatible? It contains all the valid schemas, including the ones already included in the previous version.